### PR TITLE
Touch Ups

### PR DESCRIPTION
**Pull request by davidthomas5412**
This small PR contains basic touch ups. 

We still have not been able to fix issues with the plotting. Returning the imshow and world in plotting:150 seems will likely be one of the required fixes. There are also issues with sharing subplots and sharing plots objects in the overlay plots. We will hold off committing intermittent changes to this code until we resolve all known issues. 

In the future it might be good to centralize the plotting code and take a more object oriented approach with matplotlib.

### CODE REVIEW

File: pangloss/foreground.py [hunk 1]
```diff
@@ -228,15 +231,15 @@ def plot(self,subplot=None,mag_lim=[0,24],mass_lim=[0,10**20],z_lim=[0,1.3857],f
 
         # Find the galaxies that are within the limits, and extract the useful data from them
         ra_lim, dec_lim = [ai, af], [di, df]
-        galaxies = self.return_galaxies(mag_lim,mass_lim,z_lim,ra_lim,dec_lim)
+        galaxies = self.return_galaxies(mag_lim, mass_lim, z_lim, ra_lim, dec_lim)
```
Comment (drphilmarshall) [inline pangloss/foreground.py:234]: Excellent! Love Pep8 :-)

**Comment by drphilmarshall** [comment]
Great! Are you ready for this to be merged?

File: pangloss/io.py [hunk 2]
```diff
@@ -25,6 +25,8 @@
 
         rm(filename): silent file removal
 
+        number(s): extract integer or floating point from string
```
Comment (drphilmarshall) [inline pangloss/io.py:28]: This reminds me: we should switch to numpydoc docstrings and get to work adding more. Can you issue this, please?

**Review by drphilmarshall — CHANGES_REQUESTED** [review]
Looks like you caught a few buglets as well as doing some useful lintering. Thanks! Can you please re-run all the notebooks listed in the index.ipynb and commit them as-is? They will then serve as an accurate record of our plotting failures... which then need to be issued.

File: pangloss/io.py (hunk 2 above)
Comment (davidthomas5412) [reply pangloss/io.py:28]: Added [Issue 102](https://github.com/drphilmarshall/Pangloss/issues/102)

**Comment by davidthomas5412** [comment]
Hi Phil, reran the notebooks in commit 8d161faeb515f191d759427c610efcffc17bbbdb. Should be good to go.

**Comment by drphilmarshall** [comment]
Good! Looks like we have at least one demo for Travis to run.

**Pull request merged**